Gemfile.jruby-2.6.lock.release

```diff
@@ -693,13 +693,13 @@ GEM
     msgpack (1.6.0-java)
     multi_json (1.15.0)
     multipart-post (2.2.3)
-    murmurhash3 (0.1.6-java)
+    murmurhash3 (0.1.7)
```

Comment:
I guess we're missing the pinning of murmurhash3 to 0.1.6 in the 8.6 branch?

```suggestion
    murmurhash3 (0.1.6-java)
```

Reply:
Thank. Yeah, I was planning to come early morning and check this, as I remember @andsel found this issue.

Reply:
Murmurhash3 0.1.7-java is not yet released: https://github.com/funny-falcon/murmurhash3-ruby/issues/11